spring-cloud-starter-common-security-config-web/src/test/resources/application.yml

```diff
@@ -0,0 +1,40 @@
+logging:
+  file:
+    name: sccsc-test.log
```

Comment:
Do we need to have file logging enabled?

Reply:
It was useful for finding the cyclic dependencies.

Reply:
Can it be removed now?

Reply:
I see you remove it. nm.